### PR TITLE
Add new dedicated space to manage lists of related groups

### PR DESCRIPTION
**Pull request by PushkarJ**
Fixes: https://github.com/cncf/tag-security/issues/638

/hold

### CODE REVIEW

File: partner-groups/adding-or-updating-groups.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+We welcome PRs regarding addition of new groups, when:
+1. The group is not part of the existing list
+2. The group is relevant to security
+3. Ownership of the group is shared with more than one independent entities
+4. Information such as charter, projects, artifacts of the group should be publicly accessible
+
+## Naming convention
+
+1. Alphabetical ordering is preferred
```
Comment (lumjjb) [inline partner-groups/adding-or-updating-groups.md:14]: let's `--Wall` to make this enforceable :) 
```suggestion
1. Groups named in alphabetical order
```

File: partner-groups/adding-or-updating-groups.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+2. When the name is not self-explanatory it is recommended to add a short blurb about the group
+3. Use [Title case](https://titlecaseconverter.com/) for the group name
```
Comment (lumjjb) [inline partner-groups/adding-or-updating-groups.md:16]:
```suggestion
3. Use [title case](https://titlecaseconverter.com/) for the group name
```

File: partner-groups/adding-or-updating-groups.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+We welcome PRs regarding addition of new groups, when:
+1. The group is not part of the existing list
+2. The group is relevant to security
+3. Ownership of the group is shared with more than one independent entities
+4. Information such as charter, projects, artifacts of the group should be publicly accessible
+
+## Naming convention
+
+1. Alphabetical ordering is preferred
+2. When the name is not self-explanatory it is recommended to add a short blurb about the group
```
Comment (lumjjb) [inline partner-groups/adding-or-updating-groups.md:15]:
```suggestion
2. When the name is not self-explanatory, it is recommended to add a short blurb about the group
```

File: README.md [hunk 4]
```diff
@@ -110,10 +110,8 @@ and add to list below:
 If you are new to the group, check out our [New Members Page](NEWMEMBERS.md) and submit a PR to add yourself to the [members list](#stag-members).
 
 ## Related groups
-
-* [Kubernetes Policy Working Group](https://github.com/kubernetes/community/tree/master/wg-policy)
-* [Kubernetes SIG-Auth](https://github.com/kubernetes/community/tree/master/sig-auth)
-* [NIST Big Data WG](https://bigdatawg.nist.gov/)
+- [Known partner groups](./partner-groups/README.md)
+- [Adding or Updating groups](./partner-groups/adding-or-updating-groups.md)
```
Comment (lumjjb) [inline README.md:114]: I think this goes under the governance folder, and on the `README.md` we probably should give a 1 line to briefly explain partner groups with these links as part of that description

File: .gitignore [hunk 5]
```diff
@@ -1,2 +1,3 @@
+.idea/
```
Comment (lumjjb) [inline .gitignore:1]: not quite sure what this does.

**Review by lumjjb — CHANGES_REQUESTED** [review]
Nice PR @PushkarJ !! Couple nits and comments, looks great!

File: partner-groups/README.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
+#Partner Groups
+
```
Comment (lumjjb) [inline partner-groups/README.md:2]: We probably need to talk a little bit about what partner groups are:
e.g.

- Many cloud native focused groups cover areas that may more often or not consider security, we consult for these groups, etc. create security related content.
- Groups that cover other aspects of security in detail - have groups come in and share about ... for some groups, we have representation of the partner groups in our community that give us updates during our weekly calls and inform us via issues, etc.

Maybe.. we could also add names of members that want to be a representative contact point for the various groups as well :) or we can keep that for another PR too

File: README.md (hunk 4 above)
Comment (TheFoxAtWork) [reply README.md:114]: Concur with @lumjjb .  This works better under governance.

File: partner-groups/README.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+* [IEEE P1915.1](https://standards.ieee.org/develop/project/1915.1.html): Standard for Software Defined Networking and Network Function Virtualization Security
+* [IEEE P7000](https://standards.ieee.org/develop/project/7000.html): Series of interrelated standards in development, including privacy, transparency, etc. related to ethical concerns in autonomous systems
+
+* [Kubernetes SIG-Security](https://github.com/kubernetes/community/tree/master/sig-security)
+* [Kubernetes SIG-Auth](https://github.com/kubernetes/community/tree/master/sig-auth)
+* [Kubernetes Policy Working Group](https://github.com/kubernetes/community/tree/master/wg-policy)
+
+* [NIST Computer Security Resource Center](https://csrc.nist.gov/)
+* [NIST Big Data WG](https://bigdatawg.nist.gov/)
+
```
Comment (TheFoxAtWork) [inline partner-groups/README.md:20]:
```suggestion
**CNCF**
* [CNCF TAGs](https://github.com/cncf/toc#technical-advisory-groups): Technical Advisory Groups
  * Security TAG members: (Storage - ), (App-Delivery - ), (Observability - ), (Runtime - ), (Contributor Strategy - ), (Network - )
* [CNCF TOC](https://github.com/cncf/toc): Technical Oversight Committee
  * Security TAG members: @justincormack , @lizrice 
  
**K8s**
* [Kubernetes SIG-Security](https://github.com/kubernetes/community/tree/master/sig-security)
  * Security TAG members: 
* [Kubernetes SIG-Auth](https://github.com/kubernetes/community/tree/master/sig-auth)
  * Security TAG members: 
* [Kubernetes Policy Working Group](https://github.com/kubernetes/community/tree/master/wg-policy)
  * Security TAG members: 
  
**Industry**
* [Cloud Security Alliance Working Groups](https://cloudsecurityalliance.org/research/working-groups/)
  * Security TAG members: @achetal01 
* [IEEE P2675 DevOps Security](https://standards.ieee.org/develop/project/2675.html)
  * Security TAG members: 
* [IEEE Product Safety Engineering Society](http://ewh.ieee.org/soc/pses/)
  * Security TAG members: 
* [IEEE 7009](https://standards.ieee.org/develop/project/7009.html): Standard for Fail-Safe Design of Autonomous and Semi-Autonomous Systems Working Group
  * Security TAG members: 
* [IEEE P1915.1](https://standards.ieee.org/develop/project/1915.1.html): Standard for Software Defined Networking and Network Function Virtualization Security
  * Security TAG members: 
* [IEEE P7000](https://standards.ieee.org/develop/project/7000.html): Series of interrelated standards in development, including privacy, transparency, etc. related to ethical concerns in autonomous systems
* [NIST Computer Security Resource Center](https://csrc.nist.gov/)
  * Security TAG members: 
* [NIST Big Data WG](https://bigdatawg.nist.gov/)
  * Security TAG members: 

```

File: partner-groups/adding-or-updating-groups.md (hunk 1 above)
Comment (TheFoxAtWork) [reply partner-groups/adding-or-updating-groups.md:14]:
```suggestion
1. Groups names in alphabetical order by type
```

**Review by TheFoxAtWork — COMMENTED** [review]
Changes recommended, @lumjjb had some solid ones.

**Comment by anvega** [comment]
I really like the intention of highlighting organizations engaged in security. However, I'm not sure of the usage of the word partners. A partnership implies there is a collaboration between both parties and I'm not sure that is the case for all organizations listed in the commit. Related groups is a better term, but I see partner groups and related groups as two different sections. 

Going back to #638, I do feel spotlighting and raising awareness of cross-organizational efforts has in itself more value in the context of TAG Security than the list of related organizations. Listing active efforts and parties engaged might have more relevance than an organizational directory.  

Another nit is that the readme is already fairly extensive with a lot of reading and scrolling required to get to it.

File: README.md (hunk 4 above)
Comment (PushkarJ) [reply README.md:114]: @lumjjb and @TheFoxAtWork are you both saying that the `partner-groups` directory goes under governance directory? AND Remove any reference of partner groups from repo main README and move it to governance/README.md ?

File: .gitignore (hunk 5 above)
Comment (PushkarJ) [reply .gitignore:1]: This was to ensure my Intellij IDEA files do not get committed. Can remove but suspect others may face this too.

**Comment by PushkarJ** [comment]
Tying back both of your comments @lumjjb and @anvega on if partner groups as a name makes sense, thinking of renaming the directory as `related-groups` where `partners` are a subset of those. 
Also, since it will be possible to point to a person who is liaison for a group also implies that this group has active collaboration and partnership with Security- TAG v/s others who are named without any TAG members as liaisons. Hope that works !!

File: .gitignore (hunk 5 above)
Comment (lumjjb) [reply .gitignore:1]: ah gotcha, it should be worth adding then

File: README.md (hunk 4 above)
Comment (lumjjb) [reply README.md:114]: Similar to what you have now except everything goes into `governance/partner-groups/` instead of `partner-groups/`

**Comment by PushkarJ** [comment]
@lumjjb @TheFoxAtWork @anvega made all the requested changes and removed WIP label. Please review :)

File: governance/README.md [hunk 8]
```diff
@@ -5,6 +5,7 @@ Security TAG is a [CNCF Technical Advisory Group](https://github.com/cncf/toc/tr
 * [Charter](charter.md) - mission and scope
 * [Roles](roles.md) - the work of the group is facilited by Chairs, Technical Leads, and active group members
 * [Process](process.md) - how projects are proposed and work is tracked
+* [Related Groups](./partner-groups/README.md) - List of CNCF, Kubernetes and other Industry wide groups that do related work
```
Comment (ashutosh-narkar) [inline governance/README.md:8]: Does this location exist ? Do you mean `related-groups` ?

File: governance/related-groups/README.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+  * Security TAG members: @achetal01
+* [IEEE P2675 DevOps Security](https://standards.ieee.org/develop/project/2675.html)
+  * Security TAG members:
+* [IEEE Product Safety Engineering Society](http://ewh.ieee.org/soc/pses/)
+  * Security TAG members:
+* [IEEE 7009](https://standards.ieee.org/develop/project/7009.html): Standard for Fail-Safe Design of Autonomous and Semi-Autonomous Systems Working Group
+  * Security TAG members:
+* [IEEE P1915.1](https://standards.ieee.org/develop/project/1915.1.html): Standard for Software Defined Networking and Network Function Virtualization Security
+  * Security TAG members:
+* [IEEE P7000](https://standards.ieee.org/develop/project/7000.html): Series of interrelated standards in development, including privacy, transparency, etc. related to ethical concerns in autonomous systems
```
Comment (ashutosh-narkar) [inline governance/related-groups/README.md:28]: Add `Security TAG members:`

**Review by ashutosh-narkar — COMMENTED** [review]
@PushkarJ this looks good. My only high level comment was whether it would be easier to view this if the information was documented in a table.

File: governance/README.md (hunk 8 above)
Comment (PushkarJ) [reply governance/README.md]: Good catch! Fixed.

File: governance/related-groups/README.md (hunk 9 above)
Comment (PushkarJ) [reply governance/related-groups/README.md:28]: Unless I misunderstood the comment, we actually do not know who are the TAG members for some of the groups and some of the groups may not have any representation. See comment here: https://github.com/cncf/tag-security/pull/680#issuecomment-853379794

**Comment by PushkarJ** [comment]
> @PushkarJ this looks good. My only high level comment was whether it would be easier to view this if the information was documented in a table.

Thanks for reviewing @ashutosh-narkar Yeah, thought about it that option too, I am fine with or without a table, this was just faster. If you feel strongly about it, I can convert to table in this PR, else we can punt it down for future. Let me know :)

File: partner-groups/README.md (hunk 6 above)
Comment (PushkarJ) [reply partner-groups/README.md]: The new updates should suffice to resolve this conversation.

File: governance/related-groups/README.md (hunk 9 above)
Comment (ashutosh-narkar) [reply governance/related-groups/README.md:28]: I meant you missed `Security TAG members:` on the next line.

Comment (PushkarJ) [reply governance/related-groups/README.md:28]: Aha, good catch. You have really good eyes :) Fixed it now

**Comment by ashutosh-narkar** [comment]
>Yeah, thought about it that option too, I am fine with or without a table, this was just faster. If you feel strongly about it, I can convert to table in this PR, else we can punt it down for future. Let me know :)

Thanks for making the changes @PushkarJ ! I don't have any strong opinions about the table format.

**Review by ashutosh-narkar — APPROVED** [review]

File: governance/related-groups/README.md [hunk 10]
```diff
@@ -0,0 +1,38 @@
+# Related Groups
```
Comment (lumjjb) [inline governance/related-groups/README.md:1]: nit: Sentence case

File: governance/README.md [hunk 11]
```diff
@@ -5,6 +5,7 @@ Security TAG is a [CNCF Technical Advisory Group](https://github.com/cncf/toc/tr
 * [Charter](charter.md) - mission and scope
 * [Roles](roles.md) - the work of the group is facilited by Chairs, Technical Leads, and active group members
 * [Process](process.md) - how projects are proposed and work is tracked
+* [Related Groups](./related-groups/README.md) - List of CNCF, Kubernetes and other Industry wide groups that do related work
```
Comment (lumjjb) [inline governance/README.md:8]: nit: for consistency
```suggestion
* [Related Groups](./related-groups/README.md) - list of CNCF, Kubernetes and other Industry wide groups that do related work 
```

File: governance/related-groups/README.md (hunk 10 above)
Comment (lumjjb) [reply governance/related-groups/README.md:1]: I still think we need a 1-2 sentence in this document itself if people are pointed directly to this link. 

something like "These are groups that are affiliated to the Security TAG or groups that are covering related topics relevant to the work of the TAG."

Comment (PushkarJ) [reply governance/related-groups/README.md]: Added 

> These are groups that are affiliated to or do work and cover topics relevant to the work of Security TAG

**Comment by PushkarJ** [comment]
@lumjjb Updated the PR with your suggestions. Took care of recommended lint fixes as well for files modified or added. 

Almost did the fixes for parent README.md and then found out you already have a PR for it #707 :)

File: governance/related-groups/README.md [hunk 12]
```diff
@@ -0,0 +1,51 @@
+# Related groups
+
+These are groups that are affiliated to or do work and cover topics relevant to
+the work of Security TAG
+
+## CNCF
+
+* [CNCF TAGs](https://github.com/cncf/toc#technical-advisory-groups):
+  Technical Advisory Groups
+  * Security TAG members: (Storage - ), (App-Delivery - ),
```
Comment (lumjjb) [inline governance/related-groups/README.md:10]: can we add link to each of the repos, and no need for the brackets, we can just list them verbatim

TAG-Storage, TAG-App-Delivery, etc.

File: governance/related-groups/adding-or-updating-groups.md [hunk 13]
```diff
@@ -0,0 +1,27 @@
...
+1. The name of the group changes
+2. The web hosting link, or the charter of the group changes
+3. When the group is retired
+
+## Adding a new group
+
+We welcome PRs regarding addition of new groups, when:
+
+1. The group is not part of the existing list
+2. The group is relevant to security
```
Comment (lumjjb) [inline governance/related-groups/adding-or-updating-groups.md:16]:
```suggestion
2. The group is relevant to cloud native security
```

**Review by lumjjb — COMMENTED** [review]
minor nit if not LGTM

File: governance/related-groups/README.md (hunk 12 above)
Comment (PushkarJ) [reply governance/related-groups/README.md]: Added a bulleted list with links to their repos

File: governance/related-groups/adding-or-updating-groups.md (hunk 13 above)
Comment (PushkarJ) [reply governance/related-groups/adding-or-updating-groups.md]: Fixed

File: README.md [hunk 14]
```diff
@@ -123,14 +123,6 @@ list below:
 If you are new to the group, check out our [New Members Page](NEWMEMBERS.md) and
 submit a PR to add yourself to the [members list](#stag-members).
```
Comment (lumjjb) [inline README.md:124]:
```suggestion
submit a PR to add yourself to the [members list](#stag-members).

## Related groups

There are several groups that are affiliated to or do work and cover topics relevant
 to the work of Security TAG. These can be seen [here](governance/related-groups/)

```

**Review by lumjjb — APPROVED** [review]
LGTM! Thanks @PushkarJ

**Pull request merged**